UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -652,13 +652,13 @@ <h4>Examples</h4>
 						
 					</aside>
 
-					<aside class="example" title="Conformance statements that meets minimum accessibility standards followed by detailed conformance information">
-						<p>This publication meets minimum accessibility standards.</p>
+					<aside class="example" title="Conformance statements that claims to meet minimum accessibility standards followed by detailed conformance information">
```

Comment:
I think the change here is correct but "statements" in the title attribute should be singular:

> Conformance statement that claims to meet